src/main/java/com/arcbees/bourseje/BourseJEDebug.gwt.xml

```diff
@@ -1,9 +1,12 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <module rename-to='bourseje'>
     <inherits name='com.arcbees.bourseje.BourseJE'/>
+    <inherits name="com.google.gwt.logging.Logging"/>
+
+    <add-linker name="xsiframe"/>
```

Comment:
The doc says so, but I was getting a "the iframe linker is deprecated" warning when building the app. I verified the pom and it's using GWT 2.7.